keps/sig-cloud-provider/1959-service-lb-class-field/README.md

```diff
@@ -132,16 +139,18 @@ type ServiceSpec struct {
 	// implementation is watching for Services with a matching class name. Any default load balancer
 	// implementation (e.g. cloud providers) should ignore Services that set this field.
 	// +optional
-	LoadBalancerClassName string `json:"loadBalancerClass,omitempty"`
+	LoadBalancerClass string `json:"loadBalancerClass,omitempty"`
 }
 ```
 
 * `loadBalancerClass` will be immutable only when the Service type is `LoadBalancer`, this way existing and future implementations
-do not have to worry about handling Services that change the class name. The class name is mutable and can be cleared when the
+do not have to worry about handling Services that change the class name. The class name is mutable and must be cleared when the
```

Comment:
I'm confused:

"`loadBalancerClass` will be immutable only when the Service type is `LoadBalancer` ... The class name is mutable and must be cleared when..."

Is it mutable or immutable?

Reply:
oops let me clarify.

this meant to say that the field is only mutable when the type changes from LoadBalancer and it should be cleared when that happens